adapters/slog/README.md

```diff
@@ -0,0 +1,50 @@
+# Axiom Go Adapter for slog
+
+Adapter to ship logs generated by
+[slog](https://pkg.go.dev/golang.org/x/exp/slog) to Axiom.
+
+## Quickstart
+
+Follow the [Axiom Go Quickstart](https://github.com/axiomhq/axiom-go#quickstart)
+to install the Axiom Go package and configure your environment.
+
+Import the package:
+
+```go
+// Imported as "adapter" to not conflict with the "slog" package.
+import adapter "github.com/axiomhq/axiom-go/adapters/slog"
+```
+
+You can also configure the adapter using [options](https://pkg.go.dev/github.com/axiomhq/axiom-go/adapters/slog#Option)
+passed to the [New](https://pkg.go.dev/github.com/axiomhq/axiom-go/adapters/slog#New)
+function:
+
+```go
+handler, err := adapter.New(
+    SetDataset("AXIOM_DATASET"),
+)
+```
+
+To configure the underlying client manually either pass in a client that was
+created according to the [Axiom Go Quickstart](https://github.com/axiomhq/axiom-go#quickstart)
+using [SetClient](https://pkg.go.dev/github.com/axiomhq/axiom-go/adapters/slog#SetClient)
+or pass [client options](https://pkg.go.dev/github.com/axiomhq/axiom-go/axiom#Option)
+to the adapter using [SetClientOptions](https://pkg.go.dev/github.com/axiomhq/axiom-go/adapters/slog#SetClientOptions).
+
+```go
+import adapter "github.com/axiomhq/axiom-go/axiom"
+
+// ...
+
+handler, err := adapter.New(
+    SetClientOptions(
+        axiom.SetPersonalTokenConfig("AXIOM_TOKEN", "AXIOM_ORG_ID"),
+    ),
+)
+```
+
+### ❗ Important ❗
+
+The adapter uses a buffer to batch events before sending them to Axiom. This
+buffer must be flushed explicitly by calling [Close](https://pkg.go.dev/github.com/axiomhq/axiom-go/adapters/slog#Handler.Close).
+Checkout the [example](../../examples/slog/main.go).
```

Comment:
Can we add a code example with a `func main()` that sets up the adapter, logs using slog and then closes the adapter on exit?

Reply:
Like the one that is part of this PR? 🙃

Reply:
Yeah, but not in the README

Reply:
We have them in the GoDoc. I understand your idea but that means we need to maintain three examples for each adapter (GoDo, examples folder and adapter README).

Reply:
aight